fdbserver/DeltaTree.h

```diff
@@ -899,3 +899,738 @@ struct DeltaTree {
 		return wptr - (uint8_t*)&node;
 	}
 };
+
+// ------------------------------------------------------------------
+#pragma pack(push, 1)
+template <typename T, typename DeltaT = typename T::Delta>
+struct DeltaTree2 {
```

Comment:
It would be good to add a big comment here like the original delta tree has explaining the how/why of the Partials/apply, like the original one explains Delta/apply, as it took me a decent bit of staring at the code to properly understand.

Reply:
Done.